src/ResourceManager/Sql/Commands.Sql/TransparentDataEncryption/Cmdlet/GetAzureRmSqlManagedInstanceTransparentDataEncryptionProtector.cs

```diff
@@ -0,0 +1,46 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Sql.TransparentDataEncryption.Model;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.TransparentDataEncryption.Cmdlet
+{
+    /// <summary>
+    /// Defines the Get-AzureRmSqlServerTransparentDataEncryptionProtector cmdlet
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlManagedInstanceTransparentDataEncryptionProtector", SupportsShouldProcess = true, DefaultParameterSetName = DefaultParameterSet)]
+    [Alias("Get-AzureRmSqlManagedInstanceTDEProtector")]
```

Comment:
This need to be "Get-" + ResourceManager.Common.AzureRMConstants.AzureRMPrefix +  "SqlManagedInstanceTDEProtector". Same with the other alias for set.